### PR TITLE
add new images for nginx 1.4.0, 1.5.1 for 1.25+

### PR DESCRIPTION
**Pull request by Deepzima**

### CODE REVIEW

File: modules/ingress/images.yml [hunk 1]
```diff
@@ -11,6 +11,8 @@ images:
       - "v1.1.0"
       - "v1.1.2"
       - "v1.3.1"
+      - "v1.4.0"
```
Comment (nutellinoit) [inline modules/ingress/images.yml:14]: We only need the version we want to use

**Review by nutellinoit — CHANGES_REQUESTED** [review]
Let's sync only the image we want (1.5.1)

**Comment by Deepzima** [comment]
the main difference is about the nginx version, the 1.5.1 use the  nginx version 1.21.6 instead the 1.4.0 use 1.19.10

**Review by nutellinoit — APPROVED** [review]
LGTM! Merging ~

**Comment by Deepzima** [comment]
just taked the 1.5.1

**Pull request merged**